iOS/DuckDuckGoTests/ContextualDaxDialogsFactoryTests.swift

```diff
@@ -341,6 +344,143 @@ final class ContextualDaxDialogsFactoryTests: XCTestCase {
         XCTAssertTrue(pixelReporterMock.didCallMeasureEndOfJourneyDialogDismiss)
     }
 
+    // MARK: - Manual Dismiss Dialog
+
+    func testWhenSearchResultDialogDismissCTAIsTappedThenExpectedPixelFiresAndDelegateDidTapDismissIsCalled() throws {
+        let spec = DaxDialogs.BrowsingSpec.afterSearch
+        let isShowingTryVisitSiteDialog = false
+        let result = sut.makeView(for: spec, delegate: delegate, onSizeUpdate: {})
+        let view = try XCTUnwrap(find(OnboardingFirstSearchDoneDialog.self, in: result))
+        XCTAssertFalse(pixelReporterMock.didCallMeasureSearchResultDialogDismissButtonTapped)
+        XCTAssertFalse(delegate.didCallDidTapDismissContextualOnboardingAction)
+
+
+        // WHEN
+        view.onManualDismiss(isShowingTryVisitSiteDialog)
+
+        // THEN
+        XCTAssertTrue(pixelReporterMock.didCallMeasureSearchResultDialogDismissButtonTapped)
+        XCTAssertTrue(delegate.didCallDidTapDismissContextualOnboardingAction)
+    }
```

Comment:
Super minor - Could add a //GIVEN in these tests before the setup. Unsure if intentional, and again, minor.